setup.py

```diff
@@ -15,7 +15,7 @@
     author="Andrew Edwards, Jan Brooks, James Williams",
     author_email="andrew.edwards@rackspace.com, jan.brooks@rackspace.com, james.williams@rackspace.com",
     keywords = "hier_config",
-    python_requires='~=3.6',
```

Comment:
I believe this should be changed to `>=2.7`

Reply:
This has been updated, but I guess if you comment on the delete it doesn't' reflect.